ansible/roles/common/tasks/packages.yml

```diff
@@ -3,9 +3,8 @@
 ######################### Install Basic Packages
 - name: install basic packages
   yum:
-    name: "{{ item }}"
     state: present
-  with_items: "{{common_packages}}"
+    name: "{{common_packages}}"
```

Comment:
this is good, we need to change it everywhere, `with_items` + `yum` being deprecated.
Does this work with all ansible versions ?
No other places ? Can we safely search and replace everywhere ?

( Not directly related to this "reboot vms" topic though. )

Reply:
I have not tried include_tasks. But that's a good idea. @fridim